src/Tizen.NUI/src/public/Application/NUIApplication.cs

```diff
@@ -616,6 +616,19 @@ private void EnableBorder(IBorderInterface borderInterface)
             borderEnabled = true;
             this.borderInterface = borderInterface;
         }
+
```

Comment:
Could you add a description for this method?

Reply:
thank you for review.
I added the comment.